src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetOutstandingIpos/GetOutstandingIposForCurrentPersonQueryHandlerTests.cs

```diff
@@ -0,0 +1,199 @@
+                Assert.AreEqual(_invitationWithPersonParticipant.Id, outstandingInvitationWithPersonParticipant.InvitationId);
+                Assert.AreEqual(_invitationWithPersonParticipant.Description, outstandingInvitationWithPersonParticipant.Description);
+                var outstandingInvitationWithFunctionalRoleParticipant = result.Data.Items.ElementAt(1);
+                Assert.AreEqual(_invitationWithFunctionalRoleParticipant.Id, outstandingInvitationWithFunctionalRoleParticipant.InvitationId);
+                Assert.AreEqual(_invitationWithFunctionalRoleParticipant.Description, outstandingInvitationWithFunctionalRoleParticipant.Description);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handle_ShouldReturnCorrectItems_WhenUserIsNotInAnyFunctionalRoles()
+        {
+            using (var context = new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var dut = new GetOutstandingIposForCurrentPersonQueryHandler(context, _currentUserProvider, _meApiServiceMock.Object, _plantProvider);
+                IList<string> emptyListOfFunctionalRoleCodes = new List<string>();
+                _meApiServiceMock
+                    .Setup(x => x.GetFunctionalRoleCodesAsync(TestPlant))
+                    .Returns(Task.FromResult(emptyListOfFunctionalRoleCodes));
+
+                var result = await dut.Handle(_query, default);
+
+                Assert.AreEqual(1, result.Data.Items.Count());
+                var outstandingInvitation = result.Data.Items.First();
+                Assert.AreEqual(_invitationWithPersonParticipant.Id, outstandingInvitation.InvitationId);
+                Assert.AreEqual(_invitationWithPersonParticipant.Description, outstandingInvitation.Description);
+            }
+        }
+
+        [TestMethod]
+        public async Task Handle_ShouldReturnOkResult_WhenNoCompletedIpoExists()
+        {
+            using (var context = new IPOContext(_dbContextOptions, _plantProvider, _eventDispatcher, _currentUserProvider))
+            {
+                var invitationWithPersonParticipant = context.Invitations.Single(i => i.Id == _invitationWithPersonParticipant.Id);
+
+                var invitationWithFunctionalRoleParticipant =
+                    context.Invitations.Single(i => i.Id == _invitationWithFunctionalRoleParticipant.Id);
+
+                invitationWithPersonParticipant.UnCompleteIpo(
+                    _personParticipant,
+                    _personParticipant.RowVersion.ConvertToString());
+
+                invitationWithFunctionalRoleParticipant.UnCompleteIpo(
+                    _functionalRoleParticipant,
+                    _functionalRoleParticipant.RowVersion.ConvertToString());
+
+                context.SaveChangesAsync().Wait();
+
+                var dut = new GetOutstandingIposForCurrentPersonQueryHandler(context, _currentUserProvider,
```

Comment:
split this using-block into 2 blocks:
![image](https://user-images.githubusercontent.com/19545528/114987681-b93c8900-9e95-11eb-973a-06cfc95c3208.png)

Reply:
Why?

Reply:
Most for keeping same structure as other tests. 
When I (or Henning?) first wrote like this, the thought is that the first using setup the context and ensure to save it "in DB", just as needed for the dut.Handle. The second using opens a brand new context (but it contains the same content of course)